idea/resources/intentionDescriptions/AddJvmStaticIntention/after.kt.template

```diff
@@ -0,0 +1,3 @@
+<spot>@JvmStatic</spot>
```

Comment:
The template shows a top-level method, not a method inside an `object`. Please update it to show what the intention action actually does.